app/src/main/java/com/mybus/activity/DisplayFavoritesActivity.java

```diff
@@ -29,7 +29,7 @@
 /**
  * Created by Julian Gonzalez <jgonzalez@devspark.com>
  */
-public class DisplayFavoritesActivity extends BaseDisplayActivity implements FavoriteEditListener, FavoriteDeleteListener, OnAddNewFavoriteListener, OnEditOldFavoriteListener {
+public class DisplayFavoritesActivity extends BaseDisplayActivity implements FavoriteListItemListener, FavoriteAddListener, FavoriteEditOldListener {
```

Comment:
Why do you need two listeners for edit? one is onFavoriteItemEdit inside FavoriteListItemListener and the other is FavoriteEditOldLIstener